protostar/protostar_cli.py

```diff
@@ -80,8 +81,12 @@ def __init__(
         protostar_toml_writer: ProtostarTOMLWriter,
         protostar_toml_reader: ProtostarTOMLReader,
         requester: InputRequester,
+        start_time: float = 0.0,
+        logger: Optional[Logger] = None,
```

Comment:
Why do logger is optional?

Reply:
Wouldn't be better to just pass `SilentLogger` then or something? This way you have to assert and if it away everywhere which is messy as we log a lot

Reply:
You can keep argument optional but i think you should set default then to keep property not an optional one